packages/devtools_app/lib/src/screens/memory/panes/control/secondary_controls.dart

```diff
@@ -34,153 +31,11 @@ class SecondaryControls extends StatefulWidget {
 }
 
 class _SecondaryControlsState extends State<SecondaryControls>
```

Comment:
same comment here. Let's make this stateless now

Reply:
I think they need to stay stateful, because they need context:
```
void _openSettingsDialog() {
    showDialog(
      context: context,
      builder: (context) => MemorySettingsDialog(controller),
    );
  }

```

Reply:
this call to showDialog doesn't have to be made from a Stateful widget. See example in `OpenAboutAction`

Reply:
I am afraid OpenAboutAction implements antipattern. Created discussion about it in memory space: https://chat.google.com/room/AAAA4tmxOUQ?cls=7

Reply:
Can we take the resolution out of this PR?

Reply:
Okay, it seems memory experts to not think stateless option is leak-prone. Made it stateless.